client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx

```diff
@@ -41,8 +41,23 @@ export class CredentialsForm extends Component {
 	};
 
 	handleFieldChange = event => {
+		const newForm = {
+			...this.state.form,
+			[ event.target.name ]: event.target.value,
+		};
+
+		if ( 'protocol' === event.target.name ) {
+			switch ( event.target.value ) {
+				case 'ftp':
+					newForm.port = 21;
+					break;
+				default:
+					newForm.port = 22;
+			}
```

Comment:
Are there going to be more cases? Otherwise we can just use
```js
newForm.port = 'ftp' === event.target.value ? 21 : 22;
```

Reply:
Good suggestion. Fixed in https://github.com/Automattic/wp-calypso/pull/19338/commits/c1cbcebc669508d75d04dce8589b543b87db94a6